src/ColorVectorSpace.jl

```diff
@@ -239,6 +239,7 @@ min(a::Number, b::AbstractGray) = min(promote(a,b)...)
 min(a::AbstractGray, b::Number) = min(promote(a,b)...)
 
 isfinite{T<:AbstractFloat}(c::AbstractGray{T}) = isfinite(gray(c))
+isfinite{T<:Bool}(c::AbstractGray{T}) = isfinite(gray(c))
```

Comment:
I wonder if we could just define two methods,
```julia
isfinite{T<:Normed}(c::Colorant{T}) = true
isfinite(c::Colorant) = reducec(&, isfinite, c)
```
where `reducec` would be defined similarly to [`mapc`](https://github.com/JuliaGraphics/ColorTypes.jl/blob/3fedf2023a39ec92114868a48c740f9273a2ba08/src/operations.jl#L99-L108). Obviously the same would apply for `isnan`, `isinf`.